scripts/run_mdsw.sh

```diff
@@ -23,24 +23,33 @@ DATADIR=./crashdata_mdsw_tmp
 STACKWALKER="$(getenv 'processor.command_pathname')"
 
 if [[ $# -eq 0 ]]; then
-    echo "Usage: run_mdsw.sh CRASHID"
-    exit 1
+    if [ -t 0 ]; then
+        # If stdin is a terminal, then there's no input
+        echo "Usage: run_mdsw.sh CRASHID"
+        exit 1
+    fi
+
+    # stdin is not a terminal, so pull the args from there
+    set -- ${@:-$(</dev/stdin)}
```

Comment:
This follows the same code I wrote in `scripts/process_crashes.sh`. It makes `run_mdsw.sh` sooooooo much better since you can run it against a file of crashids.